packages/dart_frog_cli/lib/src/commands/update/update.dart

```diff
@@ -43,6 +54,13 @@ class UpdateCommand extends Command<int> {
     if (isUpToDate) {
       _logger.info('$packageName is already at the latest version.');
       return ExitCode.success.code;
+    } else if (verifyOnly) {
+      _logger
+        ..info('A new version of $packageName is available.\n')
+        ..info('The latest version: $latestVersion')
```

Comment:
**Nit:**

Bold the latest version text to mimic the output from `flutter upgrade --verify-only`. I think it makes it more legible for the user.

```suggestion
        ..info(styleBold.wrap('The latest version: $latestVersion'))
```